js/sign/src/deterministic.ts

```diff
@@ -0,0 +1,55 @@
+export enum MajorType {
+  TypePosInt = 0,
```

Comment:
Can we remove the `Type` prefix from the values? `MajorType.TypeX` looks somewhat redundant.

Reply:
Done, PTAL :)